File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.cs
using System;

public class Solution 
{
    public int solution(int num, int k) 
    {
        string numStr = num.ToString();
        int answer = -1;
        for(int i=0; i<numStr.Length; i++) {
            if(numStr[i].ToString() == k.ToString()) {
                answer = i+1;
                break;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(int num, int k) 
{
    string numStr = to_string(num);
    
    size_t pos = numStr.find(to_string(k));
    if(pos == string::npos)
    {
        return -1;
    }
    else
    {    
        return pos+1;
    }
}